### ISSUE TITLE
kitchen.yml has uncommitted modifications at launch

### ISSUE THREAD

**Issue by ChefRycar** [chef-cft/bjc#375]
Encountered in 1.3.2 and 1.3.3. After running the "start me up" script and launching bjc-ecommerce in VSC, there is a .kitchen.yml (uncommitted) and .kitchen.yml.bak. The only difference is the subnet, and the uncommitted yml seems to be valid (worked fine when I tested and committed it along with the rest of the standard demo commits.)

**Comment by scarolan** [comment]
There's really no way around this if we want to make the demo available to partners and other groups who don't have access to our AWS account.  The reason it's uncommitted is because the startup script actively parses the file and replaces both subnet and security group with the *current* ones that are running in the stack.  This allows kitchen configs to be completely portable.

Suggestions for how to get around this are welcome.

**Comment by ChefRycar** [comment]
Gotcha, that makes sense and is good to know. Contemplating whether it's worth chasing a "solution" (since it doesn't break anything, just wasn't what I suspect). Will close this issue in the meantime.

**Comment by smacfarlane** [comment]
You could render out a .kitchen.local.yml that isn't tracked by git that contains the appropriate overrides.

Ex using the bjc-ecommerce cookbook:
``` 
cat .kitchen.local.yml
driver:
  security_group_ids: sg-1234567
  aws_ssh_key_id: chef_dem0
```

``` 
kitchen diagnose 
driver: 
...
  aws_ssh_key_id: chef_dem0
...
  security_group_ids: sg-1234567```

**Comment by scarolan** [comment]
Submitted a fix via #376
